### DISCUSSION TITLE
Big spaces between letters

### DISCUSSION THREAD

**Discussion by Klanor** [elfmz/far2l#2127] [Q&A]
Built and installed Far according to the intracrion.
Choose the default Ubuntu font
Got a big spaces between letters
Tried different fonts but the result is the similar for any font
![Screenshot from 2024-04-05 18-48-18](https://github.com/elfmz/far2l/assets/49516541/dcd3b595-bdae-4ceb-b035-8b194beeb03d)
Is it possible to somehow configure it?

OS: Ubuntu 22.04

**Comment by akruphi** [comment]
In my Ubuntu 22.04 I use font `Ubuntu Mono Regular` with size 14 and has not such problems. You can change font by F9->Options->Interface->Change font

**Comment by spnethw** [comment]
Touch: https://github.com/elfmz/far2l/issues/2360
